platform/ecs/task/task.go

```diff
@@ -205,9 +205,21 @@ func parseTaskArn(taskArn string) (string, error) {
 
 func (t *task) getResourceLimits() (ResourceLimits, error) {
 	var limits = ResourceLimits{}
-	subgroup := filepath.Join(ecsSubgroupName, t.id)
 
-	cgCPU, err := t.cgroup.CPU(subgroup)
+	cg, err := t.proc.Cgroup()
+	if err != nil {
+		return limits, err
+	}
+
+	memSs, ok := cg[memorySubsystem]
+	if !ok {
+		return limits, fmt.Errorf("%s subsystem not exists", memorySubsystem)
+	}
+
+	taskSG, _ := path.Split(memSs.CgroupPath) // CgroupPath: "/ecs/TASK_ID/DOCKER_ID" or "/ecs/CLUSTER_NAME/TASK_ID/DOCKER_ID"
```

Comment:
How about using path.Dir?

Reply:
Maybe path.Clean can be removed if using path.Dir.

Reply:
I see! I'll modify it to use path.Dir.